simphony_metadata.yml

```diff
@@ -323,3 +323,24 @@ CUDS_KEYS:
       default: 1.0
     CUBA.CUTOFF_DISTANCE:
       default: 1.0
+
+  COULOMB_FRICTION_FORCE:
+    parent: CUBA.FORCE
+    definition: Shear force accounting for the tangential displacement between contacting particles
+    models: [CUBA.ATOMISTIC]
+    CUBA.FRICTION_COEFFICIENT:
+      default: 0.0
+
+  DISSIPATION_FORCE:
+    parent: CUBA.FORCE
+    description: Viscous normal force describing the inelasticity of particle collisions
+    models: [CUBA.ATOMISTIC]
+    CUBA.RESTITUTION_COEFFICIENT:
+      default: 1.0
+
+  SJKR_COHESION_FORCE:
+    parent: CUBA.FORCE
+    description: Additional normal force tending to maintain the contact
```

Comment:
This one should be definition, probably remnant from the previous versions we had internally in the SSB.